.github/workflows/acceptance.yaml

```
@@ -76,36 +83,78 @@ jobs:
         run: ./gradlew test:build -x test
 
       - name: Build Mirror Node Test Docker image
-        run: docker build -t gcr.io/mirrornode/hedera-mirror-test:${{ github.base_ref }} ./test
+        run: |
+          TAG=${GITHUB_BASE_REF:-${GITHUB_REF_NAME}}
```

If you do this then you can remove the other definition of TAG. Can also put in a separate step so it's more clear.

```suggestion
          echo "TAG=${GITHUB_BASE_REF:-${GITHUB_REF_NAME}}" >> $GITHUB_ENV
```